tests/instances.tcl

```diff
@@ -111,6 +111,7 @@ proc spawn_instance {type base_port count {conf {}} {base_conf_file ""}} {
         if {$::io_threads} {
             puts $cfg "io-threads 2"
             puts $cfg "events-per-io-thread 0"
+            puts $cfg "min-io-threads-avoid-copy-reply 2"
```

Comment:
I would think we also want to set the in-string-size-avoid-copy-reply-threaded and min-string-size-avoid-copy-reply to a very small value in order to get better coverage. most tests does not use very large values.

Reply:
Oh I see since min-io-threads-avoid-copy-reply will force it on any string size it will be checked.